### DISCUSSION TITLE
When will the latest version be published to NuGet?

### DISCUSSION THREAD

**Discussion by coreagile** [bcgit/bc-csharp#576] [General]
Hi,

We are interested in using the argon2id algorithm recently introduced into this library. The code looks to be in the mainline as of a month ago, but there hasn't been a library release in some time. How often do releases happen? When do we think the argon2 code will be available in NuGet?

Thanks so much!

**Comment by peterdettman** [comment]
NuGet package 2.5.0 should be out by the end of the month; prereleases of 2.5.0 that include Argon2 are already available there. I've also just merged #570 and #571 that both update Argon2.
